src/Utils/index.js

```diff
@@ -0,0 +1,19 @@
+export const getMonthName = (index) => {
+  const months = [
```

Comment:
make this a const global variable in this file so we don't have to reinitialize it every time we call it. also rename this file to getMonths or something? or make it overall helper for time/dates and name it accordingly

Reply:
noted, fixing